src/templates/widget-admin.php

```diff
@@ -1,20 +1,16 @@
-<h3><?php echo __('General', 'mailmojo') ?></h3>
 <p>
-	<label for="<?php echo $this->get_field_id('listid') ?>">
-		<?php echo __('MailMojo List ID', 'mailmojo') ?>:
+	<label for="<?php echo $this->get_field_id('subscribeurl') ?>">
+		<?php echo __('Subscribe to Email List', 'mailmojo') ?>:
 	</label>
-	<input class="widefat" type="text"
-		id="<?php echo $this->get_field_id('listid') ?>"
-		name="<?php echo $this->get_field_name('listid') ?>"
-		value="<?php echo $instance['listid'] ?>">
-	<br/>
-	<small>
-		<?php echo __('To find the list ID: Go to the email list of your choice in MailMojo,
-		and look at the last part of the URL. That is the list ID. E.g. given
-		"mailmojo.no/lists/123", 123 is the list ID.', 'mailmojo') ?>
-	</small>
+	<select class="widefat" name="<?php echo $this->get_field_name('subscribeurl') ?>">
```

Comment:
Should have `id="<?php echo $this->get_field_id('subscribeurl') ?>"` to match the label's `for` attribute.